### ISSUE TITLE
can not change background color of .accordian-title when it is is-active 

### ISSUE THREAD

**Issue by blynnc** [zurb/foundation-sites#11797]
PROBLEM SOLVED TEMPORARILY (WORK AROUND) BY:

.is-active > .accordion-title {background-color:#CC000 !important;}
and removing the colors from global .accordian-title.

However this should be included in the styles.

 -- Leaving this work around here for reference for some others who may have this issue and need a quick fix.

CONTAINERS > ACCORDIAN
 The basics right off the 6.5.3 DOCS
https://foundation.zurb.com/sites/docs/accordion.html

I've used a bare bones Foundation 6.5.3  XY GRID set up downloaded directly from the latest build on github. I've included the CSS file in header and all the requisite foundation 6.5.3 js files and JQUERY 3.4.1 and inits. 

<div class ="grid-x">
<div class ="cell medium-12">

<ul class="accordion" data-accordion>
  <li class="accordion-item is-active" data-accordion-item>
    
    <a href="#" class="accordion-title">Accordion 1</a>

    
    <div class="accordion-content" data-tab-content>
      <p>Panel 1. Lorem ipsum dolor</p>
      <a href="#">Nowhere to Go</a>
    </div>
  </li>
  
</ul>
</div>
</div>

What should happen?

We should be able to change the background color of an .accordian-title when it is is-active
Currently it is easy to set :hover and :focus - why not is-active - This is important for usability. I looked at the javascript files and while I can see hooks for hover and focus, I don't see it for is-active. We should be able to set is-active background-color and is-active font color. 

While the PLUS and MINUS Icons are important as well (they activate on .accordian-title is-active ::before and ::after) as they do not rely on color - being able to change color as well is imperative for usability and accessibility.

This should apply to both parent and child (nested) acordian titles
without losing the ability to manipulate the color of the + - symbols as well.

...

What happens instead?

ALL the Accordian Titles (even nested)  have the same color background - because the color is set globally in .accordian.  . There is the is-not method but that does not help with this issue. 
...

## Possible Solution

...

Test Case and/or Steps to Reproduce (for bugs)
 
Don't have a test case - it is easily set up as explained above.

How to reproduce:
See above - 

 Context

I am trying to set up a basic nested accordian  CONTAINER > ACCORDIAN -

where I can change the .accordian-title background color on parent and nested acordian titles
without losing the ability to manipulate the color of the + - symbols as well.

...

Your Environment

- Foundation version(s) used: 6.5.3 Downloaded from github
- Browser(s) name and version(s): all
- Device, Operating System and version:  All
- Link to your project: 

Checklist

- [ x] I have read and follow the CONTRIBUTING.md document.
- [ ] There are no other issues similar to this one.

while there is no current issue exactly like this - I do see is-active issues report in May for accordian menus and this problem has cropped up periodically in various Foundation versions and has subsequently been remedied and then broken again.

- [x ] The issue title and template are correctly filled.

**Comment by blynnc** [comment]
I want to be a YETINAUT

**Comment by DanielRuf** [comment]
Hi @blynnc,

thanks for reporting this.
Do you want to provide a PR?

I see no check regarding this:
https://github.com/zurb/foundation-sites/blob/develop/scss/components/_accordion.scss

**Comment by blynnc** [comment]
> Hi @blynnc,
> 
> thanks for reporting this.
> Do you want to provide a PR?
> 
> I see no check regarding this:
> https://github.com/zurb/foundation-sites/blob/develop/scss/components/_accordion.scss

- WHATS A PR?   I wil admit I have not completely learned the ways of github and Morse Code that goes with it. I learn fast though. It's on my todo list to be able to do what you call PR's and othr things to make life easier on foundation.  can you explain for me how to do a PR ?

**Comment by DanielRuf** [comment]
Steps to create a PR:
* fork the repo to your profile
* create a new branch from develop branch
* give it a name for the issue that it fixes
* do the needed changes
* commit the changes
* push the local branch to your repository (fork)
* create a new PR at https://github.com/zurb/foundation-sites/pulls using your new branch

**Comment by blynnc** [comment]
I don't know where to leave this comment. Whomever is making those harassing phone calls to me after I post something on Github - consider yourself put on notice. I am going to report those. We will hunt you down and I assure you will no longer be a member of this group.

**Comment by JeremyEnglert** [comment]
> I don't know where to leave this comment. Whomever is making those harassing phone calls to me after I post something on Github - consider yourself put on notice. I am going to report those. We will hunt you down and I assure you - YOU will no longer be a member of this group or GIT HUB for that matter and you will be facing an harassment charge.

Can you send me details at hi@jem.codes? Your phone number isn't part of your profile. So I'm curious to how this is happening.

**Comment by DanielRuf** [comment]
That should never happen. At least not on GitHub where such data is not public.

I don't think and I hope that it is not directly related to GitHub.

Can you also send me details on Slack or Gitter in a DM?

**Comment by JeremyEnglert** [comment]
This problem is solved easily enough using some CSS.

`.accordion-item.is-active .accordion-title { background: red; }`

A setting would make sense, but probably not at this point in the game.
